src/wechaty_puppet/schemas/message.py

```diff
@@ -0,0 +1,161 @@
+"""
+Python Wechaty - https://github.com/wechaty/python-wechaty
+
+Authors:    Huan LI (李卓桓) <https://github.com/huan>
+            Jingjing WU (吴京京) <https://github.com/wj-Mcat>
+
+2020-now @ Copyright Wechaty
+
+Licensed under the Apache License, Version 2.0 (the 'License');
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an 'AS IS' BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+from __future__ import annotations
+from enum import Enum
+from typing import List
+from typing import Optional
+
+from dataclasses import dataclass
+
+
+class MessageType(Enum):
```

Comment:
Cloud we import all those constant values from the chatie-grpc module? Because that module is auto-generated from the chatie/grpc golden of the source repo.

Reply:
`wechaty_puppet` is a abstract module which should not contains any thing in `wechaty_puppet_**` .

There will be many `wechaty_puppet_**`, which MessageType member value may be different. eg:
```python
# wechaty_puppet_hostie
class MessageType(Enum):
    Contact = 0
    Text = 1

# wechaty_puppet_padplus
class MessageType(Enum):
    Contact = 1
    Text = 2
```
So, How to resolve different `wechaty_puppet_**` Enum constant value same as `wechaty_puppet`. 
I think keep that using names to ensure consistency is a good solution:

```python
# wechaty_puppet_chatie
MESSAGE_TYPE_MAP = {
  ChatieMessageType.Contact: MessageType.Contact
}

# wechaty_puppet_padplus
MESSAGE_TYPE_MAP = {
  PadPlusMessageType.Contact: MessageType.Contact
}
```
So, use name-convention to keep same information.

How do think?